nuxeo/client.py

```diff
@@ -309,6 +301,31 @@ def _handle_error(error):
             error = error_class.parse(error_data)
         return error
 
+    def _log_response(self, response):
+        # type: (requests.Response) -> None
+        """
+        Log the server's response based on its content type.
+
+        :param response: The server's response to handle
+        """
+
+        content_type = response.headers.get('content-type', 'application/octet-stream')
+
+        if not response.content:
+            # response.content is empty when *void_op* is True,
+            # meaning we do not want to get back what we sent
+            # or the operation does not return anything by default
+            content = '<void operation>'
```

Comment:
It could be something other than an operation, so I would just put `content = '<no content>'` or something similar.